docs/conf.py

```diff
@@ -169,8 +169,14 @@
 
 # -- Turn on nitpicky mode for sphinx (to warn about references not found) ----
 #
-# nitpicky = True
-# nitpick_ignore = []
+nitpicky = True
+intersphinx_mapping.update(
+    {
+        'astropy': ('https://docs.astropy.org/en/latest/', None),
```

Comment:
Just keep in mind that if you point to latest, you might get things that are documented but not released yet.

Reply:
yeah, i was 50/50 which to go with. i think you're right it's safer to go with `stable` now and then leave it that way.